src/main/java/tr/com/srdc/cda2fhir/jolt/AdditionalModifier.java

```diff
@@ -86,6 +86,11 @@ public static String getDisplay(Map<String, Object> map) {
 				if (medRef != null && medRef instanceof String) {
 					String mefRefString = (String) medRef;
 					Map<String, Object> resourceMap = (Map<String, Object>) temporaryContext.get("RESOURCE_MAP");
+
+					if (resourceMap == null) {
+						resourceMap = new HashMap<String, Object>();
+						temporaryContext.put("RESOURCE_MAP", resourceMap);
+					}
```

Comment:
Why did you put this here? It is meaningless. If resourceMap is empty just bail out. It means no resource has been created.

Reply:
This was my fix because a bug was created (I think because of merge conflicts) where the resourceMap was coming through null. But I fixed it so I think I can remove this now.

Reply:
Actually, considering that this will still error in the event that resourceMap is null (even if that no longer occurs) I think the null check should exist.

Reply:
Yes, null check should exists. You just should not create the RESOURCE_MAP here.